content/python/concepts/functions/functions.md

```diff
@@ -42,24 +42,27 @@ doHomework()
 
 ## Function Parameters
 
-Sometimes functions require input to provide data for their code. This input is defined using parameters.
+Functions often process data to achieve their intended task. Each value required by a function is passed as an argument when that function is called.
```

Comment:
```suggestion
Functions often process data to achieve their intended task. Each value required by a function is passed as an parameter when that function is called.
```

Reply:
The section talks about function parameters. Calling them arguments here is needlessly confusing.